currency/pairs.go

```diff
@@ -162,6 +182,18 @@ func (p Pairs) Add(pair Pair) Pairs {
 	return p
 }
 
+// GetMatch returns either the pair that is equal including the reciprocal for
+// when currencies are constructed from different exchange pairs e.g. Exchange
+// one USDT-DAI to exchange two DAI-USDT enabled/available pairs.
+func (p Pairs) GetMatch(pair Pair) (Pair, error) {
+	for x := range p {
+		if p[x].EqualIncludeReciprocal(pair) {
+			return p[x], nil
+		}
+	}
+	return Pair{}, ErrPairNotFound
+}
```

Comment:
Yes, but this one returns the pair from that pairs slice. If I have DAI-USDT from one exchange for ease of use if I need to use the book from another exchange which supports a DAI-USDT pair but it might be USDT-DAI for example it will match and return USDT-DAI.

Reply:
Thanks for clarifying! I missed that difference